Gray Code.cpp

```diff
@@ -0,0 +1,55 @@
+#include<bits/stdc++.h>
+using namespace std;
+
+void Gray_Code (int n)
+{
+  if (n <= 0)
```

Comment:
You are not following 4 space indentation.

Reply:
Spaces should be replaced by tab only right?

Reply:
2 spaces should be replaced by 4 spaces.